app/presenters/unauthorized_presenter.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+class UnauthorizedPresenter
+  def initialize(params)
+    @params = params
```

Comment:
I take that back... you can assume the asset exists because it contains a duplicate file set.